tests/data/console/controllers/fixtures/subdir/FirstFixture.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace yiiunit\data\console\controllers\fixtures\subdir;
+
+use yii\test\Fixture;
+use yiiunit\data\console\controllers\fixtures\FixtureStorage;
+
+class FirstFixture extends Fixture
+{
+    public function load()
+    {
+        FixtureStorage::$subdirFirstFixtureData[] = 'some data set for first fixture in subdir';
```

Comment:
I dont see this string being used anywhere? Shouldnt it be used in on of the unit tests?